### ISSUE TITLE
the total loss becomes NaN

### ISSUE THREAD

**Issue by xingyue-333** [kylehkhsu/tripod#1]
During the training process, the loss becomes NaN, why does this problem occur, and how can I solve it?
![image](https://github.com/user-attachments/assets/f8ffc109-f29f-4ddc-bb6c-3ccd10822387)

**Comment by kylehkhsu** [comment]
That's a rather strangely shaped loss curve with extremely high values that I don't recall seeing during our experiments. What hyperparameters are you using? If they're too extreme, it could be that e.g. the latents are completely collapsed, which would cause NaNs. It may also be helpful to view each of the loss terms separately, and to shorten the eval period and view the latent density plots.
